yarn-project/kv-store/src/interfaces/multi_map_test_suite.ts

```diff
@@ -124,12 +124,33 @@ export function describeAztecMultiMap(
     });
 
     it('should be able to delete individual values for a single key', async () => {
```

Comment:
I split this into a couple smaller tests and expended them a bit, since this is so custom